### ISSUE TITLE
Image parent is going out of bound when flicked the toolbar too hard.

### ISSUE THREAD

**Issue by iarghadip** [miladsalimiiii/android-mvvm-clean-architecture-dagger-hilt-coroutines#1]
This is what happens to the Image parent (card view) by flicking the toolbar too hard, it goes out of the screen.

![Bug 1](https://user-images.githubusercontent.com/72627807/96450528-f7812480-1233-11eb-8dc2-e6f0e8d2cd97.png)

**Comment by miladsalimiiii** [comment]
Are you implement it with my approach?Please share a **gif** from your problem if your answer is true.

**Comment by iarghadip** [comment]
No, not exactly your approach but pretty similar, can you explain how can I avoid it?

The CardView is outside of AppBarLayout, and the Space is inside RelativeLayout which is inside CollapsingToolbarLayout
